src/Microsoft.AspNet.OData.Shared/Formatter/EdmLibHelpers.cs

```diff
@@ -704,6 +704,22 @@ public static string GetClrPropertyName(IEdmProperty edmProperty, IEdmModel edmM
             return propertyName;
         }
 
+        public static ClrEnumMemberAnnotation GetClrEnumMemberAnnotation(this IEdmModel edmModel)
```

Comment:
>public static ClrEnumMemberAnnotation GetClrEnumMemberAnnotation [](start = 8, length = 64)

Wondering what is the purpose of the "public" modifier here? Looks like we don't need <summary> doc for all public methods in this internal helper class? This is up to you.